### PR TITLE
feat(search): add additional support for multi-faceted search

### PR DESCRIPTION
**Pull request by nkgentile**
Addresses #72 

Narrow search by adding facets multiple times

### CODE REVIEW

**Comment by nkgentile** [comment]
Looks like I broke something: changing the operator or picking a tag from the dropdown doesn't kick off a new request

File: src/components/SearchFacetsControl/index.tsx [hunk 1]
```diff
@@ -72,11 +69,11 @@ const SearchFacetsControl = () => {
           }
 
           if (facet) {
-            const isPresent = !!searchFacets.find(v => v.name === facet.name)
+            const disabled = !facet.operatorTypes && !!searchFacets.find(v => v.name === facet.name)
```
Comment (nkgentile) [inline src/components/SearchFacetsControl/index.tsx:72]: I tried to generalize the facets that should only be used once. Let me know if I should expand this logic.

File: package.json [hunk 2]
```diff
@@ -59,6 +59,7 @@
     "redux": "^4.0.5",
     "redux-observable": "^1.2.0",
     "rxjs": "^6.5.3",
+    "uuid": "^9.0.0",
```
Comment (robinpyon) [inline package.json:62]: Would it be possible to use `@sanity/uuid` here? (Which is already included as a dependency in `@sanity/core` and `@sanity/desk-tool`

File: src/components/SearchFacetNumber/index.tsx [hunk 3]
```diff
@@ -13,7 +13,7 @@ import SearchFacet from '../SearchFacet'
 import TextInputNumber from '../TextInputNumber'
 
 type Props = {
-  facet: SearchFacetInputNumberProps
+  facet: SearchFacetInputNumberProps & {id:string}
```
[the remark this answers is not in the document]
Comment (nkgentile) [reply src/components/SearchFacetNumber/index.tsx]: Good call! Agreed, this is much better

**Review by robinpyon — APPROVED** [review]
Thank you @nkgentile, this is great!

**Pull request merged**